src/mixins/highlightConfig.js

```diff
@@ -56,53 +56,28 @@ export default {
     },
   },
   methods: {
-    setHighlight() {
-      if (this.highlighted) {
-        this.shapeView.highlight(this.shapeBody, { highlighter: defaultHighlighter });
-        return;
-      }
-
-      this.shapeView.unhighlight(this.shapeBody, { highlighter: defaultHighlighter });
-    },
-    setErrorHighlight() {
+    setShapeHighlight() {
       if (!this.shapeView) {
         return;
       }
+      this.shapeView.unhighlight(this.shapeBody, errorHighlighter);
       if (this.hasError && this.autoValidate) {
-        this.shapeView.highlight(this.shapeBody, { highlighter: errorHighlighter });
-        return;
+        this.shapeView.highlight(this.shapeBody, errorHighlighter);
       }
-      this.shapeView.unhighlight(this.shapeBody, { highlighter: errorHighlighter });
-    },
-    setUpHighlightConfig() {
-      this.setErrorHighlight();
-      this.setHighlight();
-
-      this.shape.on('change:size', () => {
-        if (this.paperManager.hasScheduledUpdates()) {
-          this.paperManager.addOnceHandler('render:done', this.reHighlightExpandedElement);
-          return;
-        }
-        this.reHighlightExpandedElement();
-      });
-    },
-    reHighlightExpandedElement() {
+      this.shapeView.unhighlight(this.shapeBody, defaultHighlighter);
```

Comment:
Unhighlighting and then rehighlighting doesn't seem to cause any issues with flicker, etc. It seemed to be simpler and more consistent to bring this all into a single method.